prql-compiler/src/sql/dialect.rs

```diff
@@ -81,16 +81,16 @@ impl Dialect {
             | Dialect::SQLite
             | Dialect::Postgres
             | Dialect::MySql
-            | Dialect::MsSql
```

Comment:
I suppose this change stopping MSSQL tests.
https://github.com/PRQL/prql/blob/d614a5286a63f496eec1a56ba209ec2d1119e644/prql-compiler/tests/integration/main.rs#L288

Reply:
Good point. I've redone how these are handled...